src/guides/v2.3/install-gde/prereq/nginx.md

```diff
@@ -44,20 +44,20 @@ To install and configure `php-fpm`:
 1. Install `php-fpm` and `php-cli`:
 
    ```bash
-   apt-get -y install php7.2-fpm php7.2-cli
+   apt-get -y install php7.3-fpm php7.3-cli
    ```
 
    {:.bs-callout-info}
-   This command installs the latest available version of PHP 7.2.X. See [Magento 2.3.x technology stack requirements]({{ page.baseurl }}/install-gde/system-requirements-tech.html) for supported PHP versions.
+   This command installs the latest available version of PHP 7.3.X. See [Magento 2.4.x technology stack requirements]({{ page.baseurl }}/install-gde/system-requirements-tech.html) for supported PHP versions.
```

Comment:
I don't think we need to include the version of Magento in this line. Especially since the link to the tech stack page uses the page.baseurl var. It's designed to go to the same page whether you're in 2.3.x or 2.4.x docs.

Reply:
Righto